tasks/eth/base-003-holocene-fp-upgrade/.env

```diff
@@ -0,0 +1,15 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+
+# Base L1 PAO
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+# This is Base's safe but the Nested.just file uses council as a keyword and
+# we want to minimize changes
```

Comment:
Agreed here! Thanks for putting this at comment for the future readers!

Reply:
I copied the comment too 😂